apidoc/Titanium/UI/Android/Toolbar.yml

```diff
@@ -0,0 +1,255 @@
+    
+    A `Toolbar` can contain buttons, as well as certain other widgets, including text fields and labels. 
+    
+    A toolbar can be positioned everywhere in the the layout as a [View](Titanium.UI.View). It can also be used as an ActionBar 
+    for [activities](Titanium.Android.Activity). This allows a toolbar to inherit [ActionBar's](Titanium.Android.ActionBar) methods,
+    properties and events and provide a better customization of this design element. For example you can provide your own images
+    to be used as navigation button icon, nverflow menu icon and logo. In order to do that an application must be using Theme.AppCompat.NoTitleBar.
+    It can be set in the tiapp.xml file of the application.
+
+extends: Titanium.UI.View
+excludes: { properties: [ backgroundColor, backgroundDisabledColor, backgroundDisabledImage,         
+            backgroundFocusedColor, backgroundFocusedImage,
+            backgroundLeftCap, backgroundTopCap, backgroundRepeat,
+            backgroundSelectedColor, backgroundSelectedImage, layout,
+            softKeyboardOnFocus ],
+            events: [ longclick, twofingertap ] }
+since: "6.2.0"
+platforms: [android]
+
+properties:
+  - name: barColor
+    summary: Background color for the toolbar, as a color name or hex triplet.
+    description: |
+        To use a background image on the toolbar, either `barColor` must be a fully or
+        partially transparent color, 
+        or the [translucent](Titanium.UI.Android.Toolbar.translucent) property must be `true`.
+
+        For information about color values, see the "Colors" section of <Titanium.UI>. 
+    type: String
+    
+  - name: items
+    summary: An array of buttons (or other widgets) contained in the toolbar.
+    type: Array<Titanium.UI.View>
+    
+  - name: extendBackground
+    summary: If `true`, the background of the toolbar extends upwards by the height of StatusBar for current device.
+    description: |
+        This property is Numberended to be used when the toolbar is used as an ActionBar. It allows the user to specify
+        that the toolbar's background show through the status bar on Android Lollipop and above. The toolbar must be
+        placed at the top of the screen just below the status bar for this property to work as expected. This **must**
+        be specified at creation.
+    type: Boolean
+    default: Undefined. Behaves as if set to false.
+    osver: {android: {min: "21"}}
+    availability: creation
+    
+  - name: translucent
+    summary: If `true`, a translucent background color is used for the toolbar.
+    type: Boolean
+
+methods:
+  - name: collapseActionViews
+    summary: Collapses expanded ActionViews if there is any
+
+  - name: dismissPopupMenus
+    summary: Collapses expandend ActionViews and hides overflow menu
+
+  - name: getContentInsetEnd
+    summary: Returns the margin at the toolbar's content end.
+    returns:
+        type: Number
```

Comment:
Yes, I got that. 
There is no property "contentInsetEnd" in the Toolbar.yml. This is because there is no such property in the ToolbarProxy in Android.